test/main.py

```diff
@@ -0,0 +1,5 @@
+"""A very simple batch that tests basic functionality."""
+
+import hail as hl
+
+hl.init()
```

Comment:
Does this do enough to test if it works, or would we want to actually print something, like the most minor of "Hello, World" of calculations?

Reply:
At that point we're kind of testing the Hail backend -- which seems somewhat out of scope for the analysis runner.

A bigger problem is that the deployment of Hail is generally running _in parallel_. Ideally, we'd wait until the `prod deploy` finished and then run this test, but that's tricky.